src/core/datastore/utils/utils.js

```diff
@@ -1,9 +1,16 @@
 import _isEmpty from 'lodash/isEmpty';
 
+import { KinveyError } from '../../errors';
 import { isNonemptyString } from '../../utils';
 
 export const dataStoreTagSeparator = '.';
 
+export function getEntitiesPendingPushError(entityCount, prefix) {
+  const countMsg = `There are ${entityCount} entities, matching this query or id, pending push to the backend.`;
+  const errMsg = `Unable to ${prefix} on the backend, since the result might overwrite your local changes. ${countMsg}`;
```

Comment:
I would suggest to break up the error message into two sentences.

```javascript
const errMsg = `Unable to ${prefix} on the backend. The result might overwrite your local changes.`;
```

Reply:
I'm a little confused. When does `errMsg` get used? 
I think the revised version from Thomas is an improvement, but I'd like to avoid the phrase "might overwrite" if possible. Can we say something more concrete?

Reply:
`errMsg` is the entire message - it's always used. Just there were a few, slightly different, error messages (thus "prefix" parameter).

The method exists just to build the error message. As we discussed, there will someday be a general error factory, but until then, this will have to suffice.

I guess "the returned entities would overwrite your local entities" is more concrete. How about it?

Reply:
`The result will overwrite your local changes.` @tejasranade thoughts?

Reply:
Cool, I think `will` or `can` sounds more concrete. @thomasconner let's go with yours.